### ISSUE TITLE
Missing root className.

### ISSUE THREAD

**Issue by r0mm4k** [alissonmbr/react-xml-viewer#67]
Hi. Add root className to XMLViewer component. It will be very useful.

**Comment by alissonmbr** [comment]
Hey @r0mm4k, since version 2.0.0 the root element has the `rxv-container` className

**Comment by r0mm4k** [comment]
@alissonmbr sorry, I meant the props className.

**Comment by Kenan-Kuzgun** [comment]
> Hi. Add props className to XMLViewer component. It will be very useful.

Yes, I totally agree. Adding className would be essential for easier customization of the XMLViewer component. This way, we can make style and appearance changes more comfortably.

**Comment by renamartins** [comment]
It would be nice to pass down className as well as be able to change that default `rxv-container` class name to avoid conflict with existing class names in one's codebase.

**Comment by renamartins** [comment]
I opened #73 to get started on that feature if you'd like. Also, I posted another issue that might be related, if you are interested:

- https://github.com/alissonmbr/react-xml-viewer/issues/74
